e2e_playwright/st_camera_input_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+from playwright.sync_api import Page, expect
+
+from e2e_playwright.conftest import ImageCompareFunction
+
+
+@pytest.mark.only_browser("chromium")
+def test_displays_correct_number_of_elements(app: Page):
+    """Test that it renders correct number of camera_input elements."""
+    camera_input_widgets = app.get_by_test_id("stCameraInput")
+    expect(camera_input_widgets).to_have_count(2)
+
+
+@pytest.mark.only_browser("chromium")
+def test_captures_photo(app: Page):
+    """Test camera_input captures photo when 'Take photo' button clicked"""
+    # Wait for some timeout, until fake video stream available for camera_input
+    app.wait_for_timeout(2000)
+    take_photo_button = app.get_by_test_id("stCameraInputButton").first
+    take_photo_button.click(force=True)
+    app.wait_for_timeout(1000)
```

Comment:
nit: since the click triggers a rerun, it might be a better solution to use `wait_for_app_run` here, maybe with some additional timeout, e.g. `wait_for_app_run(app, wait_delay=1000)`.